.ci/update_translations_template.md

```diff
@@ -0,0 +1,11 @@
+After merging, all changes to the source language are available to translators at [Transifex](https://www.transifex.com/projects/p/cockatrice/) shortly.
+
+<br>
+
+Updated source strings for translations based on {{ .commit }}:
```

Comment:
I preferred when the {{.commit}} was its own line, this makes it easier to scan for when checking if the pr has been updated

Reply:
Hmm? It's both times one line of fixed text of the same length with an updating hash at the end?
`Updated source strings for translations based on {{ .commit }}:` vs. one additional line that says the same:
`The most recent update is based on commit: {{ .commit }}:`

I tried to reduce and combine information to make the template more light and focus on the relevant information without sacrificing useful content.

Reply:
I separated the information that targets a different group of people, the simple list of how many changes happened at the top which is easy to understand and the more technical information like the hash number and where to change this template or workflow at the bottom

Reply:
I see. Do you like the adjusted template?